examples/read_pcapng.rs

```diff
@@ -19,10 +19,12 @@ fn main() {
     let mut buf: Vec<u8> = Vec::new();
     let _ = fh.read_to_end(&mut buf);
 
-    match pcapng::block::parse_blocks(&buf[..]) {
+    match parse_blocks(buf.as_slice()) {
```

Comment:
I thought the &[..] syntax was preferred once that coersion landed? Am I missing something?